README.md

```diff
@@ -169,6 +169,7 @@ curated components, see the [Registry](https://aframe.io/aframe-registry).
 - [Video Controls](https://github.com/oscarmarinmiro/aframe-video-controls) - Video controls (play/pause/jump) for A-Frame video assets
 - [Vive Cursor](https://github.com/bryik/aframe-vive-cursor-component) - A cursor for HTC Vive controllers
 - [WebVR Controller](https://github.com/richardanaya/aframe-webvr-controller) - HTC Vive controllers
+- [WebVR-UI](https://github.com/googlevr/webvr-ui) - Enter VR Button by Google VR
```

Comment:
thanks for the PR! I'd recommend linking directly to https://github.com/googlevr/webvr-ui/blob/master/examples/aframe.html (and/or https://googlevr.github.io/webvr-ui/examples/aframe.html)